src/Regridder.jl

```diff
@@ -17,9 +17,27 @@ using Dates
 using JLD2
 
 export write_to_hdf5,
-    read_from_hdf5, dummmy_remap!, remap_field_cgll_to_rll, land_sea_mask, update_masks!, combine_surfaces!, binary_mask
+    read_from_hdf5,
+    dummmy_remap!,
+    remap_field_cgll_to_rll,
+    land_sea_mask,
+    update_masks!,
+    combine_surfaces!,
+    binary_mask,
+    nans_to_zero
 
-nans_to_zero(v) = isnan(v) ? typeof(v)(0) : v
+
+"""
+    nans_to_zero(v)
+
+Converts all NaNs to zero of the same type and throws a warning.
+
+"""
+function nans_to_zero(v)
+    v = isnan(v) ? typeof(v)(0) : v
+    @warn "NaNs were detected and converted to zeros."
```

Comment:
@charleskawczynski , @LenkaNovak I applied the suggestion in SDI #205 to throw a warning but perhaps it was not a good idea, since it is now flooding the CI logs (see this [build](https://buildkite.com/clima/climacoupler-ci/builds/901)) and we can't even see the entire log as they get truncated because too long.

Reply:
I suggest inlining/deleting this function and adding a warning if any elements are zero.

Reply:
What do you mean by inlining/deleting this function? I think those are two different things